### PR TITLE
Feature/async with managed executor

### PR DESCRIPTION
**Pull request by rainer-gepardec**

### CODE REVIEW

File: mega-zep-backend/src/main/java/com/gepardec/mega/service/model/Employee.java [hunk 1]
```diff
@@ -10,8 +10,10 @@
     private String releaseDate;
     private String workDescription;
     private Integer role;
+    private boolean active;
 
     public Employee() {
```
Comment (mml-gepardec) [inline mega-zep-backend/src/main/java/com/gepardec/mega/service/model/Employee.java]: Do we need this constructor at all?

File: mega-zep-backend/src/main/java/com/gepardec/mega/zep/service/impl/ZepServiceImpl.java [hunk 2]
```diff
@@ -1,61 +1,48 @@
...
 
 @RequestScoped
 public class ZepServiceImpl implements ZepService {
 
-    @Inject
-    Logger logger;
-
-    @Inject
-    ZepSoapPortType zepSoapPortType;
+    final Logger logger;
```
Comment (mml-gepardec) [inline mega-zep-backend/src/main/java/com/gepardec/mega/zep/service/impl/ZepServiceImpl.java]: private?

Comment (rainer-gepardec) [reply mega-zep-backend/src/main/java/com/gepardec/mega/zep/service/impl/ZepServiceImpl.java]: fixed

File: mega-zep-backend/src/main/java/com/gepardec/mega/service/model/Employee.java (hunk 1 above)
Comment (rainer-gepardec) [reply mega-zep-backend/src/main/java/com/gepardec/mega/service/model/Employee.java]: leave it for now, discuss if necessary

**Pull request merged**